include/boost/fusion/view/joint_view/joint_view.hpp

```diff
@@ -69,9 +69,8 @@ namespace boost { namespace fusion
         BOOST_CONSTEXPR BOOST_FUSION_GPU_ENABLED
         concat_last_type concat_last() const { return fusion::end(seq2); }
 
-    private:
         // silence MSVC warning C4512: assignment operator could not be generated
-        joint_view& operator= (joint_view const&);
+        BOOST_DELETED_FUNCTION(joint_view& operator= (joint_view const&))
 
         typename mpl::if_<traits::is_view<Sequence1>, Sequence1, Sequence1&>::type seq1;
```

Comment:
Watchout. You have unprotected things.

Now I need to re-review you patch to Spirit =\

Reply:
That was the only place I have found. Normally in Spirit this case did not happen.

Reply:
> Normally in Spirit this case did not happen.

It did, in two places.

Reply:
Sorry for that, I was sure that in spirit I took care of checking this when I did the pull request, I was wrong...

Reply:
Not a big problem. It actually was my fault that I did not check it then :D